packages/nestjs-trpc/lib/generators/__tests__/router.generator.spec.ts

```diff
@@ -152,6 +166,9 @@ describe('RouterGenerator', () => {
         },
       ];
 
+      procedureGenerator.generateProcedureString.mockReturnValueOnce('testQuery: publicProcedure.query(async () => "PLACEHOLDER_DO_NOT_REMOVE" as any )');
+      procedureGenerator.generateProcedureString.mockReturnValueOnce('testMutation: publicProcedure.mutation(async () => "PLACEHOLDER_DO_NOT_REMOVE" as any )');
```

Comment:
[note] The actual generation of these seemed important so I moved it to it's own test: [procedure.generator.spec.ts](https://github.com/KevinEdry/nestjs-trpc/pull/49/files#diff-65dcad7e46147205bd366f0401ed4ce9056b04f5afcb5c67833ab71baacbe791)

Feel free to resolve